src/main/java/org/snomed/snowstorm/core/data/services/AxiomConversionService.java

```diff
@@ -57,18 +57,20 @@ public SAxiomRepresentation convertAxiomMemberToAxiomRepresentation(ReferenceSet
 	}
 
 	public void populateAxiomMembers(Collection<Concept> concepts, String branchPath) {
-		AxiomRelationshipConversionService conversionService = setupConversionService(branchPath);
-		for (Concept concept : concepts) {
-			for (Axiom axiom : concept.getClassAxioms()) {
-				String owlExpression = conversionService.convertRelationshipsToAxiom(
-						mapFromInternalRelationshipType(concept.getConceptId(), axiom.getDefinitionStatusId(), axiom.getRelationships(), true));
-				axiom.setReferenceSetMember(createMember(concept, axiom, owlExpression));
-			}
-			for (Axiom gciAxiom : concept.getGciAxioms()) {
-				String owlExpression = conversionService.convertRelationshipsToAxiom(
-						mapFromInternalRelationshipType(concept.getConceptId(), gciAxiom.getDefinitionStatusId(), gciAxiom.getRelationships(), false));
-				gciAxiom.setReferenceSetMember(createMember(concept, gciAxiom, owlExpression));
+		try {
+			AxiomRelationshipConversionService conversionService = setupConversionService(branchPath);
+			for (Concept concept : concepts) {
+				for (Axiom axiom : concept.getClassAxioms()) {
+					String owlExpression = conversionService.convertRelationshipsToAxiom(mapFromInternalRelationshipType(concept.getConceptId(), axiom.getDefinitionStatusId(), axiom.getRelationships(), true));
+					axiom.setReferenceSetMember(createMember(concept, axiom, owlExpression));
+				}
+				for (Axiom gciAxiom : concept.getGciAxioms()) {
+					String owlExpression = conversionService.convertRelationshipsToAxiom(mapFromInternalRelationshipType(concept.getConceptId(), gciAxiom.getDefinitionStatusId(), gciAxiom.getRelationships(), false));
+					gciAxiom.setReferenceSetMember(createMember(concept, gciAxiom, owlExpression));
+				}
 			}
+		} catch (final ConversionException e) {
+			throw new RuntimeException("Failed to convert Relationship(s) to Axiom(s).", e);
```

Comment:
In snomed-owl-toolkit, we are throwing a ConversionException. Instead of propagating this through Snowstorm, I have wrapped the method call in a try-catch with a RuntimeException being thrown if a ConversionException is thrown.